src/Configuration/ProfileManager.cs

```diff
@@ -55,6 +65,15 @@ private static void ValidateFields(Profile profile)
                 return;
             }
 
+            if (string.IsNullOrEmpty(profile.ServerName))
+                throw new InvalidDataException();
```

Comment:
Please wrap one liners with `{ }`

Reply:
jorge, can you apply the ``` { } ``` pattern as described by @deccer ?

Reply:
I did, haven't pushed yet.  One more item to fix.

Reply:
fixed